flink-yarn/src/test/java/org/apache/flink/yarn/YarnResourceManagerDriverTest.java

```diff
@@ -627,9 +605,12 @@ protected void validateReleaseResources(Collection<YarnWorkerNode> workerNodes)
             verifyFutureCompleted(releaseAssignedContainerFuture);
         }
 
-        ContainerStatus createTestingContainerCompletedStatus(final ContainerId containerId) {
+        ContainerStatus createTestingContainerCompletedStatus(ContainerStatus containerStatus) {
             return new TestingContainerStatus(
```

Comment:
Why do we create a new `TestingContainerStatus` if we already have created a `ContainerStatus` instance `ContainerStatusPBImpl`.

Reply:
I think `ContainerStatusPBImpl` is a implementation class which contains some logic of YARN, `TestingContainerStatus` is just for test and can simplify the usage and reduce much of unnecessary check.

Reply:
Actaully, I don't know, maybe for clearfy it's used for test? I also think that we can replace `TestingContainerStatus` with `ContainerStatusPBImpl`.